.github/workflows/tests.yaml

```diff
@@ -16,7 +17,21 @@ jobs:
     container: golang:latest
     steps:
     - uses: actions/checkout@v3
+    - name: Verify Changed files
+      id: verify-changed-files
+      uses: tj-actions/verify-changed-files@v20
+      with:
+        files: |
+          !*.md
     - name: Get dependencies
+      if: steps.verify-changed-files.outputs.files_changed == 'true'
       run: go get -v -t -d ./...
     - name: Run tests
-      run: go test main_test.go main.go
+      if: steps.verify-changed-files.outputs.files_changed == 'true'
```

Comment:
We need to run tests still, only skip the coverage step please.

Reply:
it looks like it's not working either way :(

Reply:
@kilianc can you enable github actions and github pages for your fork for cover branch and try and run it.

Reply:
done: https://github.com/kilianc/awesome-go/pull/1#issuecomment-2143534773

but the action to detect changes is borked, I am going to use git directly or find a new one if that's ok